torchao/prototype/mx_formats/kernels.py

```diff
@@ -1487,6 +1487,200 @@ def triton_mx_block_rearrange(scale_tensor: torch.Tensor) -> torch.Tensor:
 
         return out
 
+    @triton.jit
+    def convert_fp32_to_fp4_packed(x_pairs):
+        """Convert FP32 pairs to packed FP4 format.
```

Comment:
Need to remove this comment since compiled artifacts does not work w/ documented triton code

Reply:
david fixed :)